### ISSUE TITLE
UV forecast

### ISSUE THREAD

**Issue by patrick-zippenfenig** [open-meteo/open-meteo#90]
Add UV forecast for the next days

**Comment by nero240399** [comment]
could I ask how it going ? 
I just tried to retrieve uv_index_max and _uv_index_max_clear_sky but all null.

**Comment by hreinnbeck** [comment]
Are you requesting for a specific model? UV is only in GFS.

**Comment by nero240399** [comment]
no I don't select any model, request it as a param in daily weather params.

**Comment by hreinnbeck** [comment]
https://api.open-meteo.com/v1/forecast?latitude=21.02&longitude=105.84&models=gfs_seamless&daily=uv_index_max,uv_index_clear_sky_max&timezone=auto

**Comment by patrick-zippenfenig** [comment]
I just saw that UV index in the asian region was indeed `null` all the time, unless `models=gfs_seamless` was used (thanks @hreinnbeck). It is now fixed with the new way GFS025 and GFS13 is used. UV index is based on UV-B radiation from GFS013. 

Technically, calculating the "biological effective dose" would be more accurate, but I did not find any radiation data for different spectrums for more than 5 days.

UV data is also available for hourly values, but  it is not explicitly listed in the documentation. Example for hourly data: https://api.open-meteo.com/v1/forecast?latitude=21.02&longitude=105.84&hourly=uv_index,uv_index_clear_sky
